primitives/version/src/lib.rs

```diff
@@ -98,6 +98,9 @@ pub mod embed;
 pub use sp_version_proc_macro::runtime_version;
 
 /// The identity of a particular API interface that the runtime might provide.
+///
+/// The id is generated by hashing the name of the runtime api with BLAKE2 using a hash size
```

Comment:
what's the name here *exactly*? Is that the trait identifier passed in the `decl_runtime_apis`? In any case, the docs would benefit from expanding on that

Reply:
Please check again